sm-backend/index.js

```diff
@@ -10,7 +10,7 @@ const db = knex({
     connection: {
         host: '127.0.0.1',
         user: 'postgres',
-        password: 'pass2database',
```

Comment:
I sorry for this... njye ntgo postgres yanjye iriho password.... niyompamvu

Reply:
ubwo nzajya nyongeramo nawe uyikuramo